docs/src/content/docs/drivers/postgres/getting-started.mdx

```diff
@@ -0,0 +1,216 @@
+
+### CLIs
+
+1. [psql](https://www.postgresql.org/docs/current/app-psql.html)
+
+### GUIs
+
+1. VSCode's [SQLTools](https://marketplace.visualstudio.com/items?itemName=mtxr.sqltools) extension
+
+### Libraries / Programmatic clients
+
+1. [Postgres.js](https://github.com/porsager/postgres/) - explicit parameter typing is required for now due to how the library is designed (by default it places the burden of calculating all type information onto the database)
+
+2. [Psycopg](https://www.psycopg.org/)
+
+### Drivers
+
+1. [JDBC](https://jdbc.postgresql.org/)
+
+## Compatibility
+
+### Supported types
+
+The definitive list of types and Postgres OIDs supported are available alongside other information in the `pg_catalog` metadata schema:
+
+<Fiddle autoLoad>
+  <Query q="SELECT typname, oid FROM pg_catalog.pg_type ORDER BY typname;" />
+</Fiddle>
+
+### Main caveats
+
+1. You can use `BEGIN READ WRITE` and `COMMIT`/`ROLLBACK` to execute multiple statements in a single atomic transaction, however transaction are fundamentally non-interactive (see [How XTDB works](/intro/data-model) for details) and this means you can't query in the middle of a transaction. All reads back to the client must happen outside of transactions.
+2. JSON-ification of results - generic Postgres drivers can't comprehensively handle the 'polymorphic' and nested types that XTDB supports internally, so many complex structures will be return using JSON columns. Given the lack of [DDL](https://en.wikipedia.org/wiki/Data_definition_language) functionality present in XTDB currently (we're [working on it](https://github.com/xtdb/xtdb/issues/3411)), it is therefore important that you control the data you insert into a given column if you want to avoid a monomorphic (simple) type being unexpectedly later returned as a JSON string because the type definition was inadvertently widened by an errant INSERT for an unrelated row. For example, if you INSERT row X with a number for column `foo` and then query it back you will see that same number, however if you then INSERT row Y with a string for column `foo` now row X will now always return column `foo` as string (i.e. it's now JSON). This widening behaviour cannot be undone using the supported XTDB APIs. Please get in touch if you run into difficulties.
+3. Lack of `SHOW` and SQL 'window functions' precludes various tools from working.
+4. Transactions do not [yet](https://github.com/xtdb/xtdb/issues/3463) return the information that is needed to correlate clients, so it is recommended to only issue write transactions from a single client for the time being (or use the HTTP API).
+5. Certain classes of error messages may be opaque to the client - it may therefore be best to debug queries using the HTTP API during the development workflow for now.
```

Comment:
Sounds like we should have a "debugging queries" page

Reply:
Good idea - noted - will follow up on this separately